scripts/mass-deletion/mass-deletion.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+if [[ -n "$GRIDKEY" || -n "$GRIDDOMAIN" ]]; then
+  echo "make sure to set env vars GRIDKEY and GRIDDOMAIN to run this script"
+  exit 1
+fi
+
+del=todelete.txt
+prog=progress.txt
+com="complete.txt"
```

Comment:
if this value is quoted, does that mean the others should be too?

Reply:
there's no spaces in the values so all are optional for quoting. I've removed the quotes from `complete` for consistency